Sources/BraintreeCore/BTAPIClient.swift

```diff
@@ -403,7 +403,21 @@ import Foundation
     func fetchAPITiming(path: String, connectionStartTime: Int?, requestStartTime: Int?, startTime: Int, endTime: Int) {
         let cleanedPath = path.replacingOccurrences(of: "/merchants/([A-Za-z0-9]+)/client_api", with: "", options: .regularExpression)
 
-        if cleanedPath != "/v1/tracking/batch/events" {
+        if cleanedPath.contains("three_d_secure") {
```

Comment:
This logic feels a bit clunky but gets us the expected results, open to alternates but seemed like the cleanest way to achieve what we needed